### PR TITLE
Fix of rake task openapi:generate

### PR DESCRIPTION
**Pull request by slemrmartin**
`InventoryCollection` without model_class can't be processed by openapi generator's task. 
Currently failing for `:service_instance_tasks`

### CODE REVIEW

**Comment by slemrmartin** [comment]
cc @pkomanek

File: lib/tasks/openapi_generate.rake [hunk 1]
```diff
@@ -56,11 +56,14 @@ class OpenapiGenerator
 
   def add_primary_reference_schema(inventory_collection)
     class_name                                                                 = inventory_collection.model_class.to_s
+    return if class_name.blank?
```
Comment (syncrou) [inline lib/tasks/openapi_generate.rake:59]: I'm not sure I'm such a fan of sprinkling conditionals in these methods.  

Is there any specs on this?

Comment (slemrmartin) [reply lib/tasks/openapi_generate.rake:59]: No there are no specs for this rake task. 
I think these generic conditions are better than condition just for `service_instance_tasks`.
Or what do you suggest?

**Comment by slemrmartin** [comment]
@syncrou I tried to add basic spec which covers these core changes, but this generator is not like the generators for public APIs.

This rake task can only update existing openapi.json from the ingress-api repo (cannot work in travis now)

So there has to be more work in separate PR than this fast fix

**Pull request merged**